### PR TITLE
Misc cleanup

### PR DESCRIPTION
**Pull request by naoyam**
Just stylistic changes. Extracted from #1772

### CODE REVIEW

File: torch/csrc/jit/codegen/cuda/scheduler/reduction_utils.h [hunk 1]
```diff
@@ -22,7 +22,7 @@ TensorView* scheduleReductionTV(
     bool has_iter_axis);
 
 // Inlining function intended for single or multi reduction fusions.
-void multiReductionInliner(
+TORCH_CUDA_CU_API void multiReductionInliner(
```
Comment (jjsjann123) [inline torch/csrc/jit/codegen/cuda/scheduler/reduction_utils.h:25]: We need to expose this because they are used in benchmarks?!

Comment (jjsjann123) [reply torch/csrc/jit/codegen/cuda/scheduler/reduction_utils.h:25]: nvm... I think tests are also built separately so these should be needed as well.
hmmm.. but how did we past upstream CI :confused:

**Review by jjsjann123 — APPROVED** [review]
LGTM. Thx for code cleaning~~~ :tada:

File: torch/csrc/jit/codegen/cuda/scheduler/reduction_utils.h (hunk 1 above)
Comment (naoyam) [reply torch/csrc/jit/codegen/cuda/scheduler/reduction_utils.h:25]: Some of these routines are used by outer-reduction tests that are not yet merged into devel.

**Pull request merged**